tests/shows.py

```diff
@@ -0,0 +1,23 @@
+import os
```

Comment:
```suggestion
# flake8: noqa: C0114, R0801
import os
```

Reply:
This should fix the linter issues by ignoring the failing checks for this test file.